src/components/editor/metaform-editor-right-drawer.tsx

```diff
@@ -47,24 +91,27 @@ const MetaformEditorRightDrawer: React.FC<Props> = ({
   };
 
   /**
-   * Updates metaform field VisiblityIf parameters
+   * Updates metaform field or section visiblityIf parameters
    *
-   * @param newMetaformField new metaform field
-   * @param eventValue FieldCondition value
+   * @param newMetaformUpdate metaform field or section
    */
-  const updateFormFieldVisiblityValues = (newMetaformField: MetaformField, eventValue: string) => {
-    setRequiredConditionFieldBoolean(eventValue);
-    setRequiredConditionInfoField(false);
-    setDisabledValue(false);
-    setRequiredConditionField("");
-    setRequiredConditionFieldBoolean("");
-    if (sectionIndex === undefined || fieldIndex === undefined) {
+  const updateFormFieldOrSectionVisiblityValues = (newMetaformUpdate: any, value: string) => {
```

Comment:
Add missing param to TSDoc. Also newMetaformUpdate could propably be more describing.